packages/react-strict-dom/src/native/stylex/index.js

```diff
@@ -632,12 +632,10 @@ export function props(
     // boxShadow
     else if (styleProp === 'boxShadow') {
       nextStyle.boxShadow = styleValue;
-      nextStyle.experimental_boxShadow = styleValue;
     }
     // filter
     else if (styleProp === 'filter') {
       nextStyle.filter = styleValue;
-      nextStyle.experimental_filter = styleValue;
     }
```

Comment:
The conditions can be removed entirely once we don't need the `experimental_` prefix.